### PR TITLE
RGB update commit

### PR DESCRIPTION
**Pull request by yiancar**
Minor RGB updates on NK65 and HS60

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/wilba_tech/wt_rgb_backlight.c [hunk 1]
```diff
@@ -1551,20 +1551,14 @@ void backlight_effect_indicators(void)
     }
 
 #if defined(RGB_BACKLIGHT_NK65)
-    if ( g_indicator_state & (1<<USB_LED_CAPS_LOCK) )
-    {
+    if ( g_indicator_state & (1<<USB_LED_CAPS_LOCK) ) {
```
Comment (yanfali) [inline keyboards/wilba_tech/wt_rgb_backlight.c]: should probably use IS_LED_ON helper

File: keyboards/wilba_tech/wt_rgb_backlight.c [hunk 2]
```diff
@@ -1551,20 +1551,14 @@ void backlight_effect_indicators(void)
...
         IS31FL3733_set_color( 7+64-1, 0, 255, 0 );
     } else {
         IS31FL3733_set_color( 7+64-1, 0, 0, 0 );
     }
-    if ( g_indicator_state & (1<<USB_LED_SCROLL_LOCK) )
-    {
+    if ( IS_LAYER_ON(1) ) {
```
Comment (yanfali) [inline keyboards/wilba_tech/wt_rgb_backlight.c]: Does this affect PCBs with Scroll and Numlock indicators?

Comment (yanfali) [reply keyboards/wilba_tech/wt_rgb_backlight.c]: Ahh, I see, this only affects the NK65?

Comment (yiancar) [reply keyboards/wilba_tech/wt_rgb_backlight.c]: Yep it basically makes the scroll lock led and the numlock led layer indicators!

**Comment by yiancar** [comment]
Better now?:)

**Comment by yiancar** [comment]
This is now ready for review/merge

**Pull request merged**